tests/include/test/macros.h

```diff
@@ -26,6 +26,12 @@
 
 #include "mbedtls/build_info.h"
 
+#if defined(__has_feature)
```

Comment:
Non-blocker: maybe this, and the identical definition from `padlock.h`, should be shared via `common.h` or maybe `build_info.h`?

Reply:
Good idea to share. I prefer to keep a separate definition of macros, though, in case we build the library and the tests with different options. So I've moved the macro definition to `test/helpers.h`.